assets/src/dashboard/components/alert/components.js

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled, { keyframes } from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import {
+  ALERT_SEVERITIES,
+  KEYBOARD_USER_SELECTOR,
+  Z_INDEX,
+} from '../../constants';
+import { TypographyPresets } from '../typography';
+
+const slideIn = keyframes`
+	from {
+		transform: translateX(100%);
+	}
+	to {
+		transform: translateX(0);
+	}
+`;
+
+const alertBackgrounds = {
+  [ALERT_SEVERITIES.ERROR]: 'danger',
+  [ALERT_SEVERITIES.WARNING]: 'warning',
+  [ALERT_SEVERITIES.INFO]: 'bluePrimary',
+  [ALERT_SEVERITIES.SUCCESS]: 'success',
+  [ALERT_SEVERITIES.DEFAULT]: 'bluePrimary',
+};
+
+const getColor = (severity = ALERT_SEVERITIES.DEFAULT) => {
```

Comment:
This might be a little more clear if it's a default provided at the `Alert` component prop level so it's apparent at the component declaration instead of looking at implementation details. That being said, not a biggie